.github/ci_commit_pins/audio.txt

```diff
@@ -1 +1 @@
-a8f4e97bd5356a7a77510cdf6a3a62e25a5dc602
\ No newline at end of file
+151ac4d85007c7e14d9ece9023a2c2b4b0cc6a40
```

Comment:
Why it needs an audio pin?

Reply:
I am trying to solve an issue for running torchrec_dlrm in torchbench, and this is required, at least when I tried it locally.